py/serialkeel/serial_keel.py

```diff
@@ -16,6 +16,26 @@
 
 Message = str
 
+def make_logger(name: str, add_formatter: bool = False) -> logging.Logger:
```

Comment:
Sorry for the misunderstanding, please move this to the test itself.
It does not need to be a part of the lib.

Reply:
sorry for misunderstanding, it's done now :+1: